File: Accounts/CAccounts/ContainerConfig.cs
using System;
using System.ComponentModel;
using System.Runtime.CompilerServices;
using Autofac;
using IContainer = Autofac.IContainer;

namespace CAccounts
{
    public static class ContainerConfig
    {
        public static IContainer Configure()
        {
            var builder = new ContainerBuilder();
            //builder.RegisterType<SalaryLoanSchedulePresenter>().As<ISalaryLoanSchedulePresenter>();
            return builder.Build();
        }

        //public static class IContainer Configure()
        //{
        //    //var builder = new ContainerBuilder();
        //    //builder.registerBuildType()
        //}
    }
}

File: Accounts/ContainerConfig.vb
Imports AATM.Accounts.PresentationLayer.Presenters
Imports Autofac

Public Module ContainerConfig

    Public Function Configure() As IContainer
        Dim builder As Autofac.ContainerBuilder = New ContainerBuilder()
        'builder.RegisterType(Of RecurringPayElementEntry)().[As](Of IRecurringPayElementView)()
        'builder.RegisterType(Of RecurringPayElementPresenter)().[As](Of ISalaryLoanSchedulePresenter)()
        Return builder.Build()
    End Function

End Module